scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/types/package.scala

```diff
@@ -53,14 +54,20 @@ package object types {
   case class Geography(wkt: String)
 
   /**
-   * Case class to serve as raw type for Json instances to distinguish them from Strings.
+   * Case class to serve as raw type for Json instances. On write, they will be transformed into
+   * Jackson JsonNode.
    *
    * See also https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#json_type
    *
    * @param wkt
    *   Well Known Text formatted string that BigQuery displays for Json
    */
-  case class Json(wkt: String)
+  case class Json(wkt: String) {
+    def asJackson: JsonNode = Json.mapper.readTree(wkt)
+  }
+  object Json {
+    private val mapper = new ObjectMapper()
+  }
```

Comment:
I'd prefer to have this in the conversion part and leave the data as dumb as possible

Reply:
Where would be the appropriate place to store the `ObjectMapper` instance, in order to not instantiate it for each row? Directly as a `private val` in `ConverterProvider`?